std/string.d

```diff
@@ -5659,52 +5659,25 @@ unittest
     });
 }
 
-
-/* ************************************************
- * Version       : v0.3
- * Author        : David L. 'SpottedTiger' Davis
- * Date Created  : 31.May.05 Compiled and Tested with dmd v0.125
- * Date Modified : 01.Jun.05 Modified the function to handle the
- *               :           imaginary and complex float-point
- *               :           datatypes.
- *               :
- * Licence       : Public Domain / Contributed to Digital Mars
```

Comment:
Well, if you think about this way: licensing in the public domain in the first place was really not possible for this person, as the over arching license for Phobos overrides the wishes of the authors of individual contributions. And if that is not true, then we would have to take this out because of the problems with the public domain mentioned above.